.azure-pipelines/common/package.yml

```diff
@@ -15,12 +15,74 @@ steps:
   displayName: 'Copy vsix to staging directory'
   inputs:
     Contents: '**/*.vsix'
-    TargetFolder: '$(build.artifactstagingdirectory)'
+    TargetFolder: '$(build.artifactstagingdirectory)/vsix'
+
+- task: CopyFiles@2
+  displayName: 'Copy web bundle to staging directory'
+  inputs:
+    Contents: 'dist/web/*.js*'
+    TargetFolder: '$(build.artifactstagingdirectory)/web/'
+
+- task: CopyFiles@2
+  displayName: 'Copy package.json to staging directory'
+  inputs:
+    Contents: 'package.json'
```

Comment:
As discussed offline, I believe we need the `resources` and `package.nls.json` as well

Reply:
Yes. I'm working on that with resources right now. I'll update these PRs once I have that figured out.